it/test/testConstants/IncomeSourceIntegrationTestConstants.scala

```diff
@@ -348,6 +351,50 @@ object IncomeSourceIntegrationTestConstants {
     }
   }
 
+  private val dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+
+  private def documentDetailsPayment( transactionId: String,
+                                      amount: BigDecimal,
+                                      clearingDate: LocalDate): JsObject= {
+    Json.obj(
+      "taxYear" -> testTaxYear,
+      "transactionId" -> transactionId,
+      "documentDescription" -> "Payment on Account",
+      "outstandingAmount" -> amount,
+      "originalAmount" -> amount,
+      "documentDueDate" -> dateFormatter.format(clearingDate),
+      "effectiveDateOfPayment" -> dateFormatter.format(clearingDate),
+      "documentDate" -> dateFormatter.format(clearingDate),
+      "paymentLot" -> "081203010025",
+      "paymentLotItem" -> "000001",
+    )
+  }
+
+  private def financialDetailsPayment( transactionId: String,
+                                       amount: BigDecimal,
+                                       clearingDate: LocalDate,
+                                       clearingSapDocument: String): JsObject = {
+    Json.obj(
+      "taxYear" -> s"$testTaxYear",
```

Comment:
I don't think we need to use string interpolation here

Reply:
In FinancialDetail, taxYear is is String (Int in DocumentDetail)